src/wayland/dmabuf/mod.rs

```diff
@@ -244,10 +245,9 @@ pub struct DmabufParamsData {
 /// A handle to a registered dmabuf global.
 ///
 /// This type may be used in equitability checks to determine which global a dmabuf is being imported to.
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy)]
```

Comment:
nit: what ordering do derive traits generally appear in?

Reply:
🤷‍♀️